src/Mvc/Mvc.Core/test/Formatters/SystemTextJsonInputFormatterTest.cs

```diff
@@ -132,6 +133,7 @@ public async Task ReadAsync_DoesNotThrowOverflowException()
     [InlineData("{\"a\":{\"b\"}}", "$.a", "'}' is invalid after a property name. Expected a ':'. Path: $.a | LineNumber: 0 | BytePositionInLine: 9.")]
     [InlineData("{\"age\":\"x\"}", "$.age", "The JSON value could not be converted to System.Decimal. Path: $.age | LineNumber: 0 | BytePositionInLine: 10.")]
     [InlineData("{\"login\":1}", "$.login", "The JSON value could not be converted to Microsoft.AspNetCore.Mvc.Formatters.SystemTextJsonInputFormatterTest+UserLogin. Path: $.login | LineNumber: 0 | BytePositionInLine: 10.")]
+    [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/40661")]
```

Comment:
If this is going to deterministically fail, we should skip it, not quarantine it.

Reply:
It may just start working again in the future if this turns out to be a runtime regression that gets fixed in an update.

Whether or not this is deterministic depends on whether you consider the behavior of the runtime team to be random 😄